elrond-wasm-debug/src/api/big_int_api_mock.rs

```diff
@@ -256,4 +256,35 @@ impl elrond_wasm::api::BigIntApi for RustBigInt {
 		let bi = BigInt::from_signed_bytes_be(bytes);
 		bi.into()
 	}
+
+	fn to_i64(&self) -> Option<i64> {
+		let (sign, digits) = self.0.to_u64_digits();
+		match sign {
+			Sign::NoSign => Some(0),
+			Sign::Plus => {
+				if digits.len() == 1 {
+					let as_u64 = digits[0];
+					if as_u64 <= i64::MAX as u64 {
+						Some(as_u64 as i64)
+					} else {
+						None
+					}
+				} else {
+					None
+				}
+			},
+			Sign::Minus => {
+				if digits.len() == 1 {
+					let as_u64 = digits[0];
+					match as_u64.cmp(&0x8000000000000000u64) {
+						Ordering::Less => Some(-(as_u64 as i64)),
```

Comment:
Wonder if there would be a way to use i64::MIN here, but with Rust's type safety, probably not

Reply:
tried it, I cannot write -i64::MIN because it "causes overflow"

Reply:
I decided for some mocks it is good enough